AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1036,6 +1037,21 @@ - (void)__setNeedsLayout
   }
 }
 
+// If not rasterized (and therefore we certainly have a view or layer),
+// Send the message to the view/layer first, as scheduleNodeForDisplay may call -displayIfNeeded.
+// Wrapped / synchronous nodes created with initWithView/LayerBlock: do not need scheduleNodeForDisplay,
```

Comment:
Okay, let's actually split this common block in half. For this one, let's use this:

// Wrapped / synchronous nodes created with initWithView/LayerBlock: do not need scheduleNodeForDisplay,
// as they don't need to display in the working range at all - since the moment they are visible, any call to
// setNeedsDisplay (triggered by the caller of __setNeedsDisplay) will sync. display at transaction commit on main.